### PR TITLE
Added turkish language pack

### PR DESCRIPTION
**Pull request by Eren5960**
- [x] This PR actually submits something - don't use this system as a messaging service!
- [x] This PR isn't duplicated - you can check if it is by scanning the small list - link is on the navigation bar for this repository.
- [x] This PR includes appropriate markdown for sections - e.g. code blocks for suggested code.
- [x] This PR description and comments in code is understandable - feel free to use your native language to write if you are not comfortable with English.
- [x] This PR has a single branch for itself in your fork - don't just commit to the master branch of your repository!
- [x] This PR has comments written in clear English - please don't write unreadable comments just for your eyes.

## Pull Request description
Added turkish language pack

## Optional information
{Type something here}

### CODE REVIEW

**Comment by Eren5960** [comment]
Okay

File: resources/config.yml [hunk 1]
```diff
@@ -1,94 +1,95 @@
...
+  # The block that the road is made of (Default is Oak Planks)
+  RoadBlock: '5:0'
+  # The block that plot walls are made of (Default is Stone Slabs)
+  WallBlock: '44:0'
+  # The block used as the plot floor (Default is Grass)
+  PlotFloorBlock: '2:0'
+  # The block that fills the rest of the plot (Default is Dirt)
+  PlotFillBlock: '3:0'
+  # The block at the bottom (Default is Bedrock)
+  BottomBlock: '7:0
```
Comment (Aericio) [inline resources/config.yml]: Oi! You deleted something.

File: resources/tur.ini [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+name.usage=/p isim <isim: dizi>
+name.success=Arsa ismi {%0} olarak ayarlandı
+
+; The /p home Subcommand
+home.name=ev
+home.alias=h
+home.desc=Arsa evini ziyaret eder
+home.usage=/p ev [id: sayı] [dünya: dizi]
+home.noplots=Arsa dünyasında değilsin
+home.notexist=Bu dünyada {%0} ID'li bir ev noktası bulunamadı
```
Comment (jasonwynn10) [inline resources/tur.ini]: Please put quotations around this translation

File: resources/tur.ini [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+home.success={%1} dünyasındaki {%0} ID'li ev noktasına ışınlandın
```
Comment (jasonwynn10) [inline resources/tur.ini]: Please put quotations around this translation

File: resources/tur.ini [hunk 4]
```diff
@@ -0,0 +1,225 @@
...
+setowner.usage=/p sahipayarla <oyuncu: dizi>
+setowner.maxplots=Bu oyuncu azami {%0} sayısına ulaştı
+setowner.success={%0} artık arsanın yeni sahibi
+
+; The /p list Subcommand
+list.name=liste
+list.alias=l
+list.desc=Oyuncuya ait arsaları listeler
+list.usage=/p liste [oyuncu: dizi]
+list.found=({%1};{%2}) da ki {%0} arsası
```
Comment (jasonwynn10) [inline resources/tur.ini]: Please put quotations around this translation

**Pull request closed without merging**